bin/node/src/chain_spec.rs

```diff
@@ -372,7 +372,7 @@ fn generate_genesis_config(
         staking: StakingConfig {
             force_era: Forcing::NotForcing,
             validator_count: authorities.len() as u32,
-            minimum_validator_count: authorities.len() as u32,
+            minimum_validator_count: 3,
```

Comment:
Could this perhaps be 4? I keep insisting that values below 4 don't really make sense for BFT algorithms. <_<"

Not a blocker though, especially if there are good reasons for "3".

Reply:
The reason is pragmatic that we have a default setup in e2e tests so that 4 validators are starting, so in order to have to remove one from the validator lists in the newly added e2e test. Better solution would be not to change this line ofc, and add   new 5fth validator for example.

Reply:
Or you could set this to 4 and start with 5 validators (if removing one is simpler than adding).

Reply:
I think not, so either we do changes to `docker-compose.yml` to spawn a new validator during tests or we add comments above that line with reasoning. Let me try if I can add Marcin node easily :)

Reply:
It's possible though it's not possible to set minimum validator count via extrinsic, hence we need to decrease it in either chain_spec.rs. Let me try this version with added new validator.

Reply:
https://paritytech.github.io/substrate/master/pallet_staking/pallet/enum.Call.html#variant.set_staking_configs

Reply:
Done